src/middlewares/one-of.ts

```diff
@@ -9,16 +9,19 @@ const dummyItem: ContextItem = { async run() {} };
 
 export type OneOfCustomMessageBuilder = (options: { req: Request }) => any;
 
+export type OneOfErrorType = 'grouped' | 'leastErroredOnly' | 'legacy';
```

Comment:
Not sure about `leastErroredOnly`, any suggestion?

Reply:
No better names yet...

Reply:
I like the name `grouped` as it describes well the behaviour.
What do you think of `flat` instead of `legacy`? It sounds exactly like the opposite of `grouped`.

As for `leastErroredOnly`... still no better ideas 😢

Reply:
`flat` sounds nice to me.